src/libs/Navigation/FreezeWrapper.tsx

```diff
@@ -28,8 +27,9 @@ function FreezeWrapper({keepVisible = false, children}: FreezeWrapperProps) {
             // if the screen is more than 1 screen away from the current screen, freeze it,
             // we don't want to freeze the screen if it's the previous screen because the freeze placeholder
             // would be visible at the beginning of the back animation then
-            if ((navigation.getState()?.index ?? 0) - (screenIndexRef.current ?? 0) > 1) {
-                InteractionManager.runAfterInteractions(() => setIsScreenBlurred(true));
+
+            if ((navigation.getState()?.index ?? 0) - (screenIndexRef.current ?? 0) >= 1) {
```

Comment:
I see that you changed to `>= 1` from `> 1`. Isn't this opposed to this comment?
```
    // we don't want to freeze the screen if it's the previous screen because the freeze placeholder
    // would be visible at the beginning of the back animation then
```

Reply:
I see that we no longer use the placeholder component. I guess this comment was not changed after the placeholder was removed. I will do it now

Reply:
@perunt this actually caused regression.
As this affects all platforms, please test and add remaining platforms, not only web.


this branch:

https://github.com/Expensify/App/assets/108292595/4cf409eb-f134-4c5a-9cd6-79c0bcdf95fd


main:

https://github.com/Expensify/App/assets/108292595/9b6ed6de-ee9d-4af6-81e9-56e649259b09

Reply:
@situchan, thanks for pointing it out. 
I retested it, and it looks like a freshly added nullish coalescing operator can let me remove this check, so I assume we can't receive navigation immediately in some cases. I'm adding the rest of the recordings.